oaknational/Oak-Web-Application
# Fix/1566 programme listing page slug

## Description

- List of changes

## Changes slug and title -> subjectSlug, subjectTitle

Fixes #1566 

## How to test

1. Go to https://deploy-preview-1567--oak-web-application.netlify.app/

Should work as intended, as this alters naming on the code base, functionality shouldn't change.

## Checklist
